src/app/static/src/app/components/outputTable/TableFilters.vue

```diff
@@ -1,3 +1,134 @@
 <template>
-    <div>Filters</div>
-</template>
\ No newline at end of file
+    <div class="form-group">
+        <h4 v-translate="'presets'"></h4>
+        <single-select :options="presetOptions"
+                       :model-value="tableSelections.preset"
+                       @update:model-value="changePresetSelection"/>
+    </div>
+    <div class="form-group">
+        <h4 v-translate="'indicator'"></h4>
+        <single-select :options="indicatorOptions"
+                       :model-value="tableSelections.indicator"
+                       @update:model-value="changeIndicatorSelection"/>
+    </div>
+    <new-filters :filters="filters"
+                 :selectedFilterOptions="tableSelections.selectedFilterOptions"
+                 @update:filters="changeFilterSelections"></new-filters>
```

Comment:
I think worth passing this by Rachel or Jeff too. I wonder if displaying this as something like "Type", "Table type", "Table" or similar might be a bit of a more using facing name than preset? Also I wonder about the titles too. Could have "Table" header with preset drop down and indicator drop down below it, then the filters? Or put indicator in the filters drop down... not really sure what is best here. 

Doesn't have to be done in this PR but I think worth getting some input from science on wording would be useful